packaging/suse/portusctl/lib/constants.rb

```diff
@@ -0,0 +1,5 @@
+# Some useful constants used by portus
+
+BUNDLER_BIN = "/srv/Portus/vendor/bundle/ruby/2.1.0/bin/bundler.ruby2.1"
+GEM_PATH    = "/srv/Portus/vendor/bundle/ruby/2.1.0/"
+HOSTNAME    = `hostname -f`.chomp
```

Comment:
I just had a better idea, we are already passing these variables when using the portusctl bash wrapper, so there's no need to set them!